input/sources/tev-000095.yaml

```diff
@@ -8,4 +20,9 @@ tgevcat_id: 95
 tgevcat_name: TeV J1713-3811
 
 papers:
-- null
+- 2016arXiv160908671H
+
+pos:
+  simbad_id: CTB 37B
```

Comment:
I think this is the wrong source:
http://simbad.u-strasbg.fr/simbad/sim-id?Ident=CTB+37B&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id

Use RX J1713, as mentioned above.